examples/CMakeLists.txt

```diff
@@ -15,13 +15,34 @@ if (TARGET MADworld)
   add_executable (t9-mad madness/t9.cc t9.impl.h)
   target_link_libraries(t9-mad LINK_PUBLIC ttg-mad ${MADNESS_DISABLEPIE_LINKER_FLAG})
 
-	#Wavefront example (shared memory)
+	#Wavefront example (workflow with shared data)
 	add_executable (wavefront-wf-mad madness/wavefront-wf.cc wavefront-wf.impl.h)
 	target_link_libraries(wavefront-wf-mad LINK_PUBLIC ttg-mad ${MADNESS_DISABLEPIE_LINKER_FLAG})
+
+  # Wavefront example (workflow with classes)
+  add_executable (wavefront-wf2-mad madness/wavefront-wf2.cc wavefront-wf2.impl.h)
```

Comment:
Are all these tests only running in the MADNESS version of the code ?

Reply:
Serialization/Deserialization is not integrated in the parsec ttg driver to this day: we only use the AM system, so it only works for POD structures. This example relies on sharedpointers and serialize/deserialize, so it cannot work with parsec driver in distributed today.